### PR TITLE
change to BSD license

### PR DESCRIPTION
**Pull request by chacalle**
## Describe changes

Update to the recommended BSD license for all UW software.

If you approve this I can go through and update all the other repos.

## Details of PR

Used these to inform the text. This will probably get updated soon with an official template from IHME. 
https://comotion.uw.edu/what-we-do/intellectual-property-licensing/open-source/#section-1-1
https://github.com/ihmeuw/ScrollyTeller/blob/master/LICENSE
https://hub.ihme.washington.edu/pages/viewpage.action?pageId=70692755

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,595 +1,11 @@
-GNU General Public License
-==========================
+Copyright (c) 2020 University of Washington. Developed at the Institute for Health Metrics and Evaluation (IHME).
```
Comment (krpaulson) [inline LICENSE.md]: Add this line? https://github.com/ihmeuw/ScrollyTeller/blob/master/LICENSE#L1

**Review by krpaulson — DISMISSED** [review]

**Comment by krpaulson** [comment]
Is there anyone else who needs to review this?

**Comment by chacalle** [comment]
> Is there anyone else who needs to review this?

I think once they finish the example license it will be clear exactly what text is needed, but I think this is fine for now.

**Comment by chacalle** [comment]
Just updated to use github to create the license text so that it will show an overview nicely

**Pull request merged**